dockers/docker-fpm-frr/supervisord.conf

```diff
@@ -63,24 +69,30 @@ autorestart=false
 startsecs=0
 stdout_logfile=syslog
 stderr_logfile=syslog
+dependent_startup=true
+dependent_startup_wait_for=rsyslogd:running
 
-[program:vtysh_b]
-command=/usr/bin/vtysh -b
```

Comment:
Why only separate this part into an extra `.conf` file?

Reply:
it depends on whether unified mode is enabled or not.